### ISSUE TITLE
Do not show focus outline after opened if parent doesn't have focus

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-context-menu#234]
The context menu should not focus the item when opened, but focus should still be inside the overlay so that when the user clicks Arrow Down, the first item gets the outline.

Related to #55
